mscore/menus.cpp

```diff
@@ -1592,6 +1592,11 @@ PalettePanel* MuseScore::newTextPalettePanel(bool defaultPalettePanel)
       stxt->setSwing(true);
       sp->append(stxt, QT_TRANSLATE_NOOP("Palette", "Swing"))->setElementTranslated(true);
 
+      stxt = new SystemText(gscore, Tid::TEMPO);
+      stxt->setXmlText(QT_TRANSLATE_NOOP("Palette", "Straight"));
```

Comment:
Could you please add [translator comments](https://doc.qt.io/qt-5/i18n-source-translation.html#translator-comments) to these strings to make it easier for translators to understand what does "Straight" mean in this context? See a similar comment for "mute"/"open" entries for example:
https://github.com/musescore/MuseScore/blob/3eb9a4da70d030cc38c8f273b7ea38a4e0c69dcd/mscore/menus.cpp#L1621-L1622

Reply:
done